### PR TITLE
Tags and theme

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Hey please review my PR for Tags and theme 

Screenshot 2022-04-15 175701

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { useTheme } from "../context/theme-content/ThemeProvider";
+
+const logoStyle = {
```
Comment (humzahasan) [inline src/components/Navbar.jsx]: Avoid inline styling

**Review by humzahasan — APPROVED** [review]
- Code changes looks good
- Good folder structure
- Like the minimalistic feel of your UI 

Good to merge!

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+.tag-close{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    padding: 2px;
+    border:0;
+    border-radius: 999px;
+    color: var(--OFF-BLACK);
```
Comment (srishtigautam21) [inline src/App.css:40]: you can use rem here

**Review by srishtigautam21 — APPROVED** [review]
The code is clean
good to merge

File: src/components/Navbar.jsx (hunk 1 above)
Comment (bhushannaikaroskar) [reply src/components/Navbar.jsx]: done

File: src/App.css (hunk 2 above)
Comment (bhushannaikaroskar) [reply src/App.css:40]: done

**Pull request merged**